### ISSUE TITLE
_Graphics_math_float_impl::init_scale initializes a translate matrix instead of a scale matrix

### ISSUE THREAD

**Issue by mikekazakov** [mikebmcl/P0267_RefImpl#72]
xgraphicsmathfloat_impl.h:228 contains a typo:
```
		inline typename _Graphics_math_float_impl::matrix_2d_data_type _Graphics_math_float_impl::init_translate(const typename _Graphics_math_float_impl::point_2d_data_type& val) noexcept {
			return create_matrix_2d(1.0f, 0.0f, 0.0f, 1.0f, val._X, val._Y);
		}
		inline typename _Graphics_math_float_impl::matrix_2d_data_type _Graphics_math_float_impl::init_scale(const typename _Graphics_math_float_impl::point_2d_data_type& val) noexcept {
			return create_matrix_2d(1.0f, 0.0f, 0.0f, 1.0f, val._X, val._Y);
		}
```

**Comment by AaronBallman** [comment]
I can confirm. This seems to be the root cause of why arc drawing is broken -- when I fixed the scaling matrix to no longer be a translation matrix, suddenly my circles were appropriately sized.

**Comment by mikebmcl** [comment]
Resolved by 9b3237e9
